[rospy.client][INFO] 2020-10-30 00:31:22,586: init_node, name[/rostful_server], pid[3549]
[xmlrpc][INFO] 2020-10-30 00:31:22,586: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 00:31:22,586: Started XML-RPC server [http://freight1207:48327/]
[rospy.init][INFO] 2020-10-30 00:31:22,586: ROS Slave URI: [http://freight1207:48327/]
[rospy.impl.masterslave][INFO] 2020-10-30 00:31:22,586: _ready: http://freight1207:48327/
[xmlrpc][INFO] 2020-10-30 00:31:22,587: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-30 00:31:22,587: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-30 00:31:22,686: registered with master
[rospy.rosout][INFO] 2020-10-30 00:31:22,687: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 00:31:22,688: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 00:31:22,689: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 00:31:22,693: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 00:31:22,982: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-10-30 00:31:23,752: topic[/enable_software_runstop] adding connection to [http://freight1207:48327/], count 0
[rospy.internal][INFO] 2020-10-30 00:31:23,753: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rosout][INFO] 2020-10-30 00:31:23,772: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-30 00:31:23,801: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][INFO] 2020-10-30 00:32:28,840: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-30 00:32:28,841: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-30 00:32:28,843: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 00:32:28,843: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-30 00:32:28,843: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-30 00:32:28,843: topic[/enable_software_runstop] removing connection to http://freight1207:48327/
[rospy.impl.masterslave][INFO] 2020-10-30 00:32:28,844: Closing
[rospy.internal][ERROR] 2020-10-30 00:32:28,845: Unable to initiate TCP/IP socket to freight1207:54574 (http://freight1207:48327/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.core][INFO] 2020-10-30 00:32:29,007: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-30 00:34:15,327: init_node, name[/rostful_server], pid[5546]
[xmlrpc][INFO] 2020-10-30 00:34:15,327: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 00:34:15,328: Started XML-RPC server [http://freight1207:38529/]
[rospy.init][INFO] 2020-10-30 00:34:15,328: ROS Slave URI: [http://freight1207:38529/]
[rospy.impl.masterslave][INFO] 2020-10-30 00:34:15,328: _ready: http://freight1207:38529/
[rospy.registration][INFO] 2020-10-30 00:34:15,328: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2020-10-30 00:34:15,328: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-10-30 00:34:15,428: registered with master
[rospy.rosout][INFO] 2020-10-30 00:34:15,428: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 00:34:15,430: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 00:34:15,431: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 00:34:15,435: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 00:34:15,709: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-10-30 00:34:16,494: topic[/enable_software_runstop] adding connection to [http://freight1207:38529/], count 0
[rospy.internal][INFO] 2020-10-30 00:34:16,495: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:34:16,552: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][INFO] 2020-10-30 00:34:16,588: rostful_server: Started server on port 8080
[rosout][INFO] 2020-10-30 00:36:34,517: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-30 00:36:34,518: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-30 00:36:34,520: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 00:36:34,520: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-30 00:36:34,520: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-30 00:36:34,520: topic[/enable_software_runstop] removing connection to http://freight1207:38529/
[rospy.impl.masterslave][INFO] 2020-10-30 00:36:34,520: Closing
[rospy.internal][ERROR] 2020-10-30 00:36:34,521: Unable to initiate TCP/IP socket to freight1207:38810 (http://freight1207:38529/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.core][INFO] 2020-10-30 00:36:34,624: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-30 00:36:37,211: init_node, name[/rostful_server], pid[6152]
[xmlrpc][INFO] 2020-10-30 00:36:37,211: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 00:36:37,211: Started XML-RPC server [http://freight1207:38865/]
[rospy.init][INFO] 2020-10-30 00:36:37,211: ROS Slave URI: [http://freight1207:38865/]
[rospy.impl.masterslave][INFO] 2020-10-30 00:36:37,211: _ready: http://freight1207:38865/
[xmlrpc][INFO] 2020-10-30 00:36:37,212: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-30 00:36:37,212: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-30 00:36:37,312: registered with master
[rospy.rosout][INFO] 2020-10-30 00:36:37,312: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 00:36:37,314: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 00:36:37,314: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 00:36:37,319: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 00:36:37,590: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-30 00:36:38,364: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-30 00:36:38,379: topic[/enable_software_runstop] adding connection to [http://freight1207:38865/], count 0
[rospy.internal][INFO] 2020-10-30 00:36:38,380: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:36:38,395: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][INFO] 2020-10-30 00:37:36,657: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-30 00:37:36,657: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-30 00:37:36,659: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 00:37:36,659: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-30 00:37:36,660: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-30 00:37:36,660: topic[/enable_software_runstop] removing connection to http://freight1207:38865/
[rospy.impl.masterslave][INFO] 2020-10-30 00:37:36,660: Closing
[rospy.internal][ERROR] 2020-10-30 00:37:36,660: Unable to initiate TCP/IP socket to freight1207:33257 (http://freight1207:38865/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.core][INFO] 2020-10-30 00:37:36,824: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-30 00:37:40,419: init_node, name[/rostful_server], pid[7911]
[xmlrpc][INFO] 2020-10-30 00:37:40,419: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 00:37:40,420: Started XML-RPC server [http://freight1207:59588/]
[rospy.init][INFO] 2020-10-30 00:37:40,420: ROS Slave URI: [http://freight1207:59588/]
[rospy.impl.masterslave][INFO] 2020-10-30 00:37:40,420: _ready: http://freight1207:59588/
[xmlrpc][INFO] 2020-10-30 00:37:40,420: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-30 00:37:40,420: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-30 00:37:40,520: registered with master
[rospy.rosout][INFO] 2020-10-30 00:37:40,520: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 00:37:40,522: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 00:37:40,523: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 00:37:40,527: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 00:37:40,726: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,586: topic[/current/pose] adding connection to [http://freight1207:48294/], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,587: topic[/remote_control] adding connection to [http://freight1207:59588/], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,589: topic[/current/pose] adding connection to [http://freight1207:59588/], count 1
[rospy.internal][INFO] 2020-10-30 00:37:41,590: topic[/enable_software_runstop] adding connection to [http://freight1207:59588/], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,591: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,591: topic[/local_control/cancel] adding connection to [http://freight1207:59588/], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,592: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,593: topic[/local_control/initialpose] adding connection to [http://freight1207:59588/], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,593: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,594: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,595: topic[/local_control/goto] adding connection to [http://freight1207:59588/], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,595: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,595: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 00:37:41,622: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][INFO] 2020-10-30 00:37:41,696: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-30 00:37:41,788: topic[/local_control/initialpose] adding connection to [/local_control_initialpose], count 1
[rospy.internal][INFO] 2020-10-30 00:37:41,789: topic[/local_control/cancel] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-30 00:37:41,790: topic[/local_control/goto] adding connection to [/local_control_navigation], count 1
[rospy.internal][INFO] 2020-10-30 00:37:41,790: topic[/remote_control] adding connection to [/remote_control], count 1
[rosout][WARNING] 2020-10-30 00:37:44,446: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: 0.5
    locationY: -0.19
    orientation: 0.49
  - 
    locationX: 0.45
    locationY: -1.23
    orientation: -0.71 to topic /local_control/goto
[rospy.core][INFO] 2020-10-30 01:24:27,004: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-10-30 01:24:27,009: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:24:27,009: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-30 01:24:27,009: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:24:27,009: topic[/local_control/goto] removing connection to /local_control_navigation
[rospy.internal][INFO] 2020-10-30 01:24:27,010: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-30 01:24:27,010: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:24:27,010: topic[/local_control/initialpose] removing connection to /local_control_initialpose
[rospy.internal][INFO] 2020-10-30 01:24:27,010: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:24:27,010: topic[/remote_control] removing connection to /remote_control
[rospy.internal][INFO] 2020-10-30 01:24:27,013: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:24:27,013: topic[/local_control/cancel] removing connection to /local_control_navigation
[rospy.internal][ERROR] 2020-10-30 01:24:27,014: Unable to initiate TCP/IP socket to freight1207:54124 (http://freight1207:59588/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:24:27,014: Unable to initiate TCP/IP socket to freight1207:54124 (http://freight1207:59588/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:24:27,014: Unable to initiate TCP/IP socket to freight1207:54124 (http://freight1207:59588/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:24:27,014: Unable to initiate TCP/IP socket to freight1207:54124 (http://freight1207:59588/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:24:27,014: Unable to initiate TCP/IP socket to freight1207:54124 (http://freight1207:59588/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:24:27,015: Unable to initiate TCP/IP socket to freight1207:54124 (http://freight1207:59588/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/enable_software_runstop] removing connection to http://freight1207:59588/
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/local_control/goto] removing connection to http://freight1207:59588/
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/local_control/initialpose] removing connection to http://freight1207:59588/
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/remote_control] removing connection to http://freight1207:59588/
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/local_control/cancel] removing connection to http://freight1207:59588/
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/current/pose] removing connection to http://freight1207:48294/
[rospy.internal][INFO] 2020-10-30 01:24:27,015: topic[/current/pose] removing connection to http://freight1207:59588/
[rospy.impl.masterslave][INFO] 2020-10-30 01:24:27,015: atexit
[rospy.client][INFO] 2020-10-30 01:26:17,112: init_node, name[/rostful_server], pid[16909]
[xmlrpc][INFO] 2020-10-30 01:26:17,112: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 01:26:17,113: Started XML-RPC server [http://freight1207:44795/]
[rospy.init][INFO] 2020-10-30 01:26:17,113: ROS Slave URI: [http://freight1207:44795/]
[rospy.impl.masterslave][INFO] 2020-10-30 01:26:17,113: _ready: http://freight1207:44795/
[xmlrpc][INFO] 2020-10-30 01:26:17,113: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-30 01:26:17,113: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-30 01:26:17,213: registered with master
[rospy.rosout][INFO] 2020-10-30 01:26:17,213: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 01:26:17,215: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 01:26:17,215: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 01:26:17,220: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 01:26:17,423: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-30 01:26:18,275: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-30 01:26:18,280: topic[/enable_software_runstop] adding connection to [http://freight1207:44795/], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,281: topic[/current/pose] adding connection to [http://freight1207:44795/], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,282: topic[/remote_control] adding connection to [http://freight1207:44795/], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,283: topic[/local_control/cancel] adding connection to [http://freight1207:44795/], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,284: topic[/local_control/initialpose] adding connection to [http://freight1207:44795/], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,285: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,285: topic[/local_control/goto] adding connection to [http://freight1207:44795/], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,286: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,286: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,286: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,287: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,287: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:26:18,341: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][INFO] 2020-10-30 01:26:25,906: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-30 01:26:25,906: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-30 01:26:25,910: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:26:25,910: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-30 01:26:25,910: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:26:25,910: topic[/rosout] removing connection to /rosout
[rospy.internal][ERROR] 2020-10-30 01:26:25,911: Unable to initiate TCP/IP socket to freight1207:47492 (http://freight1207:44795/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:26:25,911: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-30 01:26:25,911: Unable to initiate TCP/IP socket to freight1207:47492 (http://freight1207:44795/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:26:25,911: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:26:25,911: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-30 01:26:25,912: Unable to initiate TCP/IP socket to freight1207:47492 (http://freight1207:44795/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:26:25,912: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-30 01:26:25,912: Unable to initiate TCP/IP socket to freight1207:47492 (http://freight1207:44795/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:26:25,912: topic[/enable_software_runstop] removing connection to http://freight1207:44795/
[rospy.internal][ERROR] 2020-10-30 01:26:25,913: Unable to initiate TCP/IP socket to freight1207:47492 (http://freight1207:44795/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:26:25,913: topic[/local_control/goto] removing connection to http://freight1207:44795/
[rospy.internal][ERROR] 2020-10-30 01:26:25,913: Unable to initiate TCP/IP socket to freight1207:47492 (http://freight1207:44795/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:26:25,913: topic[/local_control/initialpose] removing connection to http://freight1207:44795/
[rospy.internal][INFO] 2020-10-30 01:26:25,913: topic[/remote_control] removing connection to http://freight1207:44795/
[rospy.internal][INFO] 2020-10-30 01:26:25,913: topic[/local_control/cancel] removing connection to http://freight1207:44795/
[rospy.internal][INFO] 2020-10-30 01:26:25,913: topic[/current/pose] removing connection to http://freight1207:44795/
[rospy.impl.masterslave][INFO] 2020-10-30 01:26:25,913: Closing
[rospy.core][INFO] 2020-10-30 01:26:26,077: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-30 01:27:26,647: init_node, name[/rostful_server], pid[17028]
[xmlrpc][INFO] 2020-10-30 01:27:26,647: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 01:27:26,648: Started XML-RPC server [http://freight1207:51421/]
[rospy.init][INFO] 2020-10-30 01:27:26,648: ROS Slave URI: [http://freight1207:51421/]
[rospy.impl.masterslave][INFO] 2020-10-30 01:27:26,648: _ready: http://freight1207:51421/
[xmlrpc][INFO] 2020-10-30 01:27:26,648: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-30 01:27:26,648: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-30 01:27:26,748: registered with master
[rospy.rosout][INFO] 2020-10-30 01:27:26,748: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 01:27:26,750: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 01:27:26,751: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 01:27:26,755: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 01:27:26,996: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-30 01:27:27,810: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-30 01:27:27,814: topic[/current/pose] adding connection to [http://freight1207:51421/], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,816: topic[/remote_control] adding connection to [http://freight1207:51421/], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,817: topic[/enable_software_runstop] adding connection to [http://freight1207:51421/], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,818: topic[/local_control/cancel] adding connection to [http://freight1207:51421/], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,819: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,820: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,820: topic[/local_control/initialpose] adding connection to [http://freight1207:51421/], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,821: topic[/local_control/goto] adding connection to [http://freight1207:51421/], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,821: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,822: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,822: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,823: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:27:27,823: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
[rosout][WARNING] 2020-10-30 01:28:10,412: publishing header: 
  seq: 0
  stamp: 
    secs: 0
    nsecs:         0
  frame_id: ''
goals: 
  - 
    locationX: 0.5
    locationY: -0.19
    orientation: 0.49
  - 
    locationX: 0.45
    locationY: -1.23
    orientation: -0.71 to topic /local_control/goto
[rosout][INFO] 2020-10-30 01:28:13,512: rostful_server: Shutting down the server
[rospy.core][INFO] 2020-10-30 01:28:13,512: signal_shutdown [Closing]
[rospy.internal][INFO] 2020-10-30 01:28:13,516: topic[/enable_software_runstop] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:28:13,516: topic[/enable_software_runstop] removing connection to /software_runstop
[rospy.internal][INFO] 2020-10-30 01:28:13,516: topic[/local_control/goto] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:28:13,517: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-10-30 01:28:13,517: topic[/local_control/initialpose] removing connection to /rostful_server
[rospy.internal][INFO] 2020-10-30 01:28:13,517: topic[/remote_control] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-30 01:28:13,517: Unable to initiate TCP/IP socket to freight1207:44324 (http://freight1207:51421/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:28:13,518: Unable to initiate TCP/IP socket to freight1207:44324 (http://freight1207:51421/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:28:13,518: topic[/local_control/cancel] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-30 01:28:13,518: Unable to initiate TCP/IP socket to freight1207:44324 (http://freight1207:51421/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2020-10-30 01:28:13,518: Unable to initiate TCP/IP socket to freight1207:44324 (http://freight1207:51421/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:28:13,519: topic[/current/pose] removing connection to /rostful_server
[rospy.internal][ERROR] 2020-10-30 01:28:13,519: Unable to initiate TCP/IP socket to freight1207:44324 (http://freight1207:51421/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:28:13,519: topic[/enable_software_runstop] removing connection to http://freight1207:51421/
[rospy.internal][ERROR] 2020-10-30 01:28:13,519: Unable to initiate TCP/IP socket to freight1207:44324 (http://freight1207:51421/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 580, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2020-10-30 01:28:13,519: topic[/local_control/goto] removing connection to http://freight1207:51421/
[rospy.internal][INFO] 2020-10-30 01:28:13,519: topic[/local_control/initialpose] removing connection to http://freight1207:51421/
[rospy.internal][INFO] 2020-10-30 01:28:13,519: topic[/remote_control] removing connection to http://freight1207:51421/
[rospy.internal][INFO] 2020-10-30 01:28:13,519: topic[/local_control/cancel] removing connection to http://freight1207:51421/
[rospy.internal][INFO] 2020-10-30 01:28:13,520: topic[/current/pose] removing connection to http://freight1207:51421/
[rospy.impl.masterslave][INFO] 2020-10-30 01:28:13,520: Closing
[rospy.core][INFO] 2020-10-30 01:28:13,651: signal_shutdown [atexit]
[rospy.client][INFO] 2020-10-30 01:34:27,432: init_node, name[/rostful_server], pid[17599]
[xmlrpc][INFO] 2020-10-30 01:34:27,432: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-10-30 01:34:27,433: Started XML-RPC server [http://freight1207:49529/]
[rospy.init][INFO] 2020-10-30 01:34:27,433: ROS Slave URI: [http://freight1207:49529/]
[rospy.impl.masterslave][INFO] 2020-10-30 01:34:27,433: _ready: http://freight1207:49529/
[xmlrpc][INFO] 2020-10-30 01:34:27,433: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-10-30 01:34:27,433: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-10-30 01:34:27,533: registered with master
[rospy.rosout][INFO] 2020-10-30 01:34:27,533: initializing /rosout core topic
[rospy.rosout][INFO] 2020-10-30 01:34:27,535: connected to core topic /rosout
[rospy.simtime][INFO] 2020-10-30 01:34:27,536: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][WARNING] 2020-10-30 01:34:27,540: rostful_server: Delaying the start 1 seconds
[rospy.internal][INFO] 2020-10-30 01:34:27,782: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2020-10-30 01:34:28,595: rostful_server: Started server on port 8080
[rospy.internal][INFO] 2020-10-30 01:34:28,600: topic[/remote_control] adding connection to [http://freight1207:49529/], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,601: topic[/current/pose] adding connection to [http://freight1207:49529/], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,602: topic[/enable_software_runstop] adding connection to [http://freight1207:49529/], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,603: topic[/local_control/cancel] adding connection to [http://freight1207:49529/], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,604: topic[/remote_control] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,605: topic[/local_control/initialpose] adding connection to [http://freight1207:49529/], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,605: topic[/current/pose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,606: topic[/local_control/goto] adding connection to [http://freight1207:49529/], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,606: topic[/enable_software_runstop] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,606: topic[/local_control/cancel] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,607: topic[/local_control/initialpose] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,607: topic[/local_control/goto] adding connection to [/rostful_server], count 0
[rospy.internal][INFO] 2020-10-30 01:34:28,628: topic[/enable_software_runstop] adding connection to [/software_runstop], count 1
